sklearn/linear_model/tests/test_sgd.py

```diff
@@ -254,20 +254,33 @@ def test_plain_has_no_average_attr(klass):
     clf = klass(average=True, eta0=.01)
     clf.fit(X, Y)
 
-    assert hasattr(clf, 'average_coef_')
-    assert hasattr(clf, 'average_intercept_')
-    assert hasattr(clf, 'standard_intercept_')
-    assert hasattr(clf, 'standard_coef_')
+    assert hasattr(clf, '_average_coef')
+    assert hasattr(clf, '_average_intercept')
+    assert hasattr(clf, '_standard_intercept')
+    assert hasattr(clf, '_standard_coef')
 
     clf = klass()
     clf.fit(X, Y)
 
-    assert not hasattr(clf, 'average_coef_')
-    assert not hasattr(clf, 'average_intercept_')
-    assert not hasattr(clf, 'standard_intercept_')
-    assert not hasattr(clf, 'standard_coef_')
+    assert not hasattr(clf, '_average_coef')
+    assert not hasattr(clf, '_average_intercept')
+    assert not hasattr(clf, '_standard_intercept')
+    assert not hasattr(clf, '_standard_coef')
 
 
+# TODO: remove in 0.25
+@pytest.mark.parametrize('klass', [SGDClassifier, SGDRegressor])
```

Comment:
we tend to use `estimator` to parametrize over estimators

Reply:
The other tests in the same file use 'klass'; I'd rather keep uniform.